openapi/definitions.yaml

```diff
@@ -271,6 +271,14 @@ EnrichedMessage:
     - fiscal_code
     - created_at
     - sender_service_id
+    - service_name
+    - organization_name
+    - message_title
+PublicMessage:
+  x-one-of: true
+  allOf:
+    - $ref: "#/CreatedMessageWithoutContent" 
+    - $ref: "#/EnrichedMessage" 
```

Comment:
Should it be `PublicMessage`?

Reply:
Well... we pass from a function that is `retrievedToPublicMessage` so I think it could be...